nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/test/java/org/nuxeo/ecm/platform/picture/core/test/TestPictureResize.java

```diff
@@ -71,6 +73,7 @@ public void closeOperationContext() {
 
     @Test
     public void testResizer() throws Exception {
+        assumeFalse(SystemUtils.IS_OS_MAC);
```

Comment:
Could you rewrite the commit comment to explain why you remove this test for Mac, if this is because of mime type detection that doesn't work on Mac then create another ticket for the record.

Reply:
haha my bad it is already the case https://jira.nuxeo.com/browse/NXP-27622 :+1: